docs/source/environments/braintumor.rst

```diff
@@ -0,0 +1,163 @@
+- The T2 region corresponds to areas of moderate tumor cell density, typically >16% of the carrying capacity
+
+Using these thresholds, our model is capable of generating virtual MRI predictions allowing us to track T1 and T2 tumor radii dynamically over time and spatially define dose regions during radiation therapy administration.
+
+To simulate the full treatment cycle–diagnosis, therapy, and follow-up–we allow the virtual tumor to grow until the T1 region reaches a radius of 10-20mm, triggering the onset of therapy. We then apply radiation according to a predefined schedule, followed by a post-treatment simulation period of 100 days of free growth.
+
+
+Numerical Implementation
+------------------------
+
+
+The external beam radiation therapy schedule follows the standard protocol adopted by the University of Washington Medical Center, and is divided into two sequential stages:
+
+- Stage 1: 28 days of 1.8 :math:`(Gy)` each, delivered to the T2 region + a 25 mm margin
+- Stage 2: 6 days of 1.8 :math:`(Gy)` each, delivered to the T1 region + a 20 mm margin
+
+This results in a total dose of 61.2 :math:`(Gy)` delivered over 34 days of treatment.
+Radiation is administered on a 5-days-on, 2-days-off schedule to account for weekend breaks.
+
+
+Nondimensional Model
+------------------------
+
+We derive the numerical implementation scheme for those looking for inner details of the environment. We use a first-order finite-difference scheme to approxiate the dimensionless PDE:
+
+.. math::
+  \bar{c}_t(\bar{x}, \bar{t}) &= D^* \nabla^2 \bar{c} +  \bar{c} - R^*(\alpha, \beta, d(\bar{x}, \bar{t}), \bar{t}) \bar{c} \\
+
+Consider the first-order taylor approximation as
+
+.. math::
+  \bar{c}(\bar{x}, \bar{t}+1) = \bar{c}(\bar{x}, \bar{t}) + \Delta \bar{t} \cdot \bar{c}_t(\bar{x}, \bar{t})
+
+with finite spatial derivatives approximated by first-order centered differences
+
+.. math::
+  \frac{\partial^2 \bar{c}}{\partial \bar{x}^2} = \frac{\bar{c}_{j+1}^n - 2\bar{c}_j^n+\bar{c}_{j-1}^n}{(\Delta \bar{x})^2}
+
+where :math:`\Delta \bar{t}=d\bar{t}=\text{dimensionless time step}`, :math:`\Delta \bar{x}=d\bar{x}=\text{dimensionless spatial step}`, :math:`n=0, ..., Nt`, :math:`j=0, ..., Nx`, where :math:`Nt` and :math:`Nx` are the total number of discretized temporal and spatial steps respectively. Substituting into our original equation yields
+
+.. math::
+  \bar{c}_{j}^{n+1} = \bar{c}_{j}^{n} + \Delta \bar{t} (D^* \cdot (\frac{\bar{c}_{j+1}^n - 2\bar{c}_j^n+\bar{c}_{j-1}^n}{(\Delta \bar{x})^2}) + \bar{c}_{j}^n - R^* \bar{c}_{j}^n)
+
+The last thing to consider is the boundary conditions for finding :math:`\bar{c}_{j}^{n+1}` when :math:`j = 0` or :math:`j = Nx`. In these cases, we set :math:`\bar{c}_{-1}^{n} = \bar{c}_{1}^{n}` and :math:`\bar{c}_{Nx+1}^{n} = \bar{c}_{Nx-1}^{n}` respectively to create a symmetric and mirrored concentration field across the boundary to satisfy the no-flux boundary condition.
+
+References
+------------------------
+
```

Comment:
Add online links for the bibliography when applicable. [4] Is not a formal citaiton without some details.